eigen: jacobiSvd is deprecated

**Avogadro version: (please complete the following information from the About box):**
 - Avogadrolibs: 9e08b9b51c37091775df71d30b08fc68c4a77b46
 - Qt: 6.10.0

**Desktop version: (please complete the following information):**
 - OS: EndeavourOS
 - Compiler: GCC 15.2.1
 - Eigen3: 5.0.1

**Describe the bug**

```
$ make
…
core/molecule.cpp:1042:20: warning: 'jacobiSvd' is
      deprecated: Options should be specified using method's template parameter. [-Wdeprecated-declarations]
 1042 |   auto svd = coord.jacobiSvd(Eigen::ComputeThinU | Eigen::ComputeThinV);
      |                    ^
/usr/include/eigen3/Eigen/src/Core/MatrixBase.h:377:42: note: 'jacobiSvd' has been explicitly marked deprecated
      here
  377 |   inline JacobiSVD<PlainObject, Options> jacobiSvd(unsigned int computationOptions) const;
      |                                          ^
core/molecule.cpp:1042:20: warning: 'jacobiSvd<0>'
      is deprecated: Options should be specified using method's template parameter. [-Wdeprecated-declarations]
 1042 |   auto svd = coord.jacobiSvd(Eigen::ComputeThinU | Eigen::ComputeThinV);
      |                    ^
/usr/include/eigen3/Eigen/src/Core/MatrixBase.h:376:3: note: 'jacobiSvd<0>' has been explicitly marked deprecated
      here
  376 |   EIGEN_DEPRECATED_WITH_REASON("Options should be specified using method's template parameter.")
      |   ^
/usr/include/eigen3/Eigen/src/Core/util/Macros.h:965:62: note: expanded from macro 'EIGEN_DEPRECATED_WITH_REASON'
  965 | #define EIGEN_DEPRECATED_WITH_REASON(message) __attribute__((deprecated(message)))
      |                                                              ^
2 warnings generated.
```

**To Reproduce**

Build avogadrolibs.

**Expected behavior**
No deprecation warnings.

--- a/avogadro/core/molecule.cpp
+++ b/avogadro/core/molecule.cpp
@@ -1039,7 +1039,8 @@ std::pair<Vector3, Vector3> Molecule::bestFitPlane(const Array<Vector3>& pos)
   coord.colwise() -= centroid;
 
   // we only need the left-singular matrix
-  auto svd = coord.jacobiSvd(Eigen::ComputeThinU | Eigen::ComputeThinV);
+  Eigen::JacobiSVD<MatrixX> svd(coord,
+                                Eigen::ComputeThinU | Eigen::ComputeThinV);
   Vector3 plane_normal = svd.matrixU().rightCols<1>();
 
   return std::make_pair(centroid, plane_normal);

